.env.example

```diff
@@ -15,3 +15,7 @@ CONTRACT_ADDRESS="0x2854CFaC53FCaB6C95E28de8C91B96a31f0af8DD"
 E2E_FAUCET_ADDRESS="0xaF9053bB6c4346381C77C2FeD279B17ABAfCDf4d"
 # Path to proto descriptor for celestia-prover
 PROTO_DESCRIPTOR_PATH="./provers/celestia-prover/proto_descriptor.bin"
+# Auth token for the celestia node
+export CELESTIA_NODE_AUTH_TOKEN=""
+# Namespace of the rollup on celestia
+export CELESTIA_NAMESPACE="0f0f0f0f0f0f0f0f0f0f"
```

Comment:
we may need the script to actually load these env variables using dotenv but I can rebase my PR on top of this to see if it resolves.

Ref: https://github.com/celestiaorg/celestia-zkevm-ibc-demo/pull/95/files#diff-a24dadc55401d7adb38b60de42eef753f235017bcedd6c81d2ee2bb46cbe4390R11